packages/icons/stories/icon.doc.stories.mdx

```diff
@@ -0,0 +1,95 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Icon, AddDocumentIcon } from "@jpmorganchase/uitk-icons";
+import { withFlexGap } from "docs/decorators/withFlexGap";
+import { ComponentAnatomy } from "docs/components/ComponentAnatomy";
+
+import {
+  CharacteristicUsage,
+  CSSClassTable,
+} from "css-variable-docgen-components";
+
+<Meta
+  title="Documentation/Icons/Icon"
+  component={Icon}
+  parameters={{
+    viewMode: "docs",
+  }}
+/>
+
+# Icon
+
+Icon is an important visual representation of an application, a capability, or some other concept or specific entity with meaning on a interface. It is expected to help user to recognize and understand a content or an action regardless languages or culture boundaries.
+
+If an icon is interactive, use [Button](?path=/docs/documentation-core-button--button-variants) (`CTA`, `Primary` or `Secondary` variant).
+
+<Canvas>
+  <Story id="icons-icon--toolkit-icon" />
+</Canvas>
+
+## Use Icon when...
+
+- You need a visual representation of a concept.
+- An icon doesn't need to be interactive.
+
+## Features
+
+We have a default icon set that starts from 12px for HD & MD, then 14px for LD, 16px for TD. (Since our base size of HD/default is 10px, but we don’t allowing icon size that is smaller then 12px, hence why here, the HD/default will start from 12px, and its 2X is 20px, 3X is 30px etc.)
+Have more size choices by adding multiplies to 2x, 3x, 4x etc.
+
+<!--  <Canvas><Story id="" /></Canvas> -->
+
+We provide 2 types of icons: default and solid. Some special icon may excluded from it like plus and loader
+
+<Canvas>
+  <Story id="icons-icon--toolkit-types" />
+</Canvas>
+
+### Icons List
+
+<Canvas>
+  <Story id="icons-icon--all-icons-with-search" />
+</Canvas>
+
+## Configuring Icon
+
+### API
+
+```
+import { Icon } from "@jpmorganchase/uitk-icons";
+```
+
+### Props
+
+<ArgsTable of={Icon} />
+
+## Anatomy
+
+We have default and solid icons.  
+You can supply custom icons as `children` of the `Icon` component, by using `<svg>`.
```

Comment:
This doesn't fall under Anatomy to me.
Also I believe the custom icon guidance needs to change after the other PR.